Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -110,7 +108,7 @@
   - RUDE
   - SAD
   - SANITARY
-  - SCALY
+  # - SCALY Could this be speciest? just need someones opinion
```

Comment:
sentient anthro lizards are not real

Reply:
specism ss14

Reply:
once again, sentient anthro lizards are not real

Reply:
> sentient anthro lizards are not real

not with that attitude

Reply:
Sentient anthro reptiles are quite real in 14.

Reply:
Reptilians :trollface: